code/modules/surgery/organs/internal/stomach/stomach_ethereal.dm

```diff
@@ -3,11 +3,19 @@
 	icon_state = "stomach-p" //Welp. At least it's more unique in functionaliy.
 	desc = "A crystal-like organ that stores the electric charge of ethereals."
 	organ_traits = list(TRAIT_NOHUNGER) // We have our own hunger mechanic.
-	///basically satiety but electrical
-	var/crystal_charge = ETHEREAL_CHARGE_FULL
+	/// Where the energy of the stomach is stored.
+	var/obj/item/stock_parts/cell/cell
 	///used to keep ethereals from spam draining power sources
 	var/drain_time = 0
 
+/obj/item/organ/internal/stomach/ethereal/Initialize(mapload)
+	. = ..()
+	cell = new/obj/item/stock_parts/cell/ethereal(src)
```

Comment:
Do we want this in the contents of the stomach?

It means it'll be included in `get_all_contents` checks which in means stuff like "Electrical Activity" from botany would affect it. (And MAYBE EMPs?) Might be safer to have it be `nullspace`d

Reply:
Do I just put null in the parameter?